ding/data/buffer/middleware/priority.py

```diff
@@ -54,7 +54,10 @@ def __init__(
 
     def push(self, chain: Callable, data: Any, meta: Optional[dict] = None, *args, **kwargs) -> BufferedData:
         if meta is None:
-            meta = {'priority': self.max_priority}
+            if 'priority' in data:
+                meta = {'priority': data['priority'].item()}
+            else:
```

Comment:
add unittest for this new if-else branch

Reply:
fixed